docs/source/release.rst

```diff
@@ -13,6 +13,7 @@ Release Notes
 * :feature:`2126` Add translation rules for isnull() and notnull() for pyspark backend
 * :feature:`2062` Implement read_csv for omniscidb backend
 * :feature:`2097` Date, DateDiff and TimestampDiff implementations for OmniSciDB
+* :bug:`2170` Fix millisecond issue for OmniSciDB #2167, MySQL #2169, PostgreSQL #2166 and Pandas #2168 backends
```

Comment:
add then the same as ``:bug:`2170` `` otherwise these don't render

Reply:
done! I used `:issue:` for that. thanks!